release-information/release-notes/6_0_0/6_0_0-release-notes-core-changes.md

```diff
@@ -16,6 +16,20 @@ The following new features were added for the {{site.productname}} 6.0 release.
 
 ### Feature name
 
+### Print plugin moved to TinyMCE Core
```

Comment:
I noticed while doing the docs for the `hr` move that this is in the wrong spot... this isn't a new feature it's a functionality change so it should be under that section. I've reworked the section for my PR to follow the same major and minor layout used for features/enhancements.

- [ ] Move to correct section

Reply:
Matched what you did with HR

Reply:
Wait, no, that was the other file

Reply:
Okay, have moved both now